.devcontainer/Dockerfile

```diff
@@ -1,8 +1,39 @@
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.234.0/containers/dotnet/.devcontainer/base.Dockerfile
 
+### Install .NET 6.0 SDK
+# copied from https://github.com/microsoft/vscode-dev-containers/blob/9ead9dfe607ac62c408180d25a71d499c0876e39/containers/dotnet/.devcontainer/base.Dockerfile
 # [Choice] .NET version: 6.0, 5.0, 3.1, 6.0-bullseye, 5.0-bullseye, 3.1-bullseye, 6.0-focal, 5.0-focal, 3.1-focal
-ARG VARIANT="6.0-bullseye"
-FROM mcr.microsoft.com/vscode/devcontainers/dotnet:0-${VARIANT}
```

Comment:
I like that we no longer depend on an intermediate image and that each step has it's own chapter for what it needs when installing. :D